src/components/auth/LogoContainer.css

```diff
@@ -0,0 +1,42 @@
+.LoginContainer .LogoContainer {
```

Comment:
You are defining the CSS of the `LogoContainer`, is there a reason why you're defining the CSS to be applied to it only inside of the `LoginContainer`?
You can use css modules if you like too, that will ensure that the CSS defined here and used later in the component will be used only for the components importing it.